subjects/ascii-art-reverse.en.md

```diff
@@ -0,0 +1,42 @@
+## ascii-art-reverse
+
+### Objectives
+
+Ascii-art-reverse consists on reversing the process, convertin the graphic representation into a text.
+
+- You will have to create a `txt` file with a graphic representation of a random phrase;
```

Comment:
The semicolon at the end of the lines in the list is very formal, and I'm a little allergic to semicolons now but as you wish ^^